enterprise/models.py

```diff
@@ -289,6 +289,11 @@ class Meta:
         help_text=_("Specifies whether to allow access to the saml configuration screen in the admin portal")
     )
 
+    enable_universal_link = models.BooleanField(
+        default=False,
+        help_text=_("Specifies whether to allow univeral link generation in the admin portal.")
```

Comment:
I think this help text might be a bit misleading? As is, I read this help text as controlling whether universal link generation appears in the admin portal "Settings" page (an ECS driven setting), but in reality, it's whether the customer has universal link generation switched on/off within the UI.

Maybe adding a disclaimer that this field is controlled by the customer's selection in the UI versus controlled by internal staff? 🤔